src/iface/interface/mod.rs

```diff
@@ -396,6 +396,12 @@ impl Interface {
     /// This function returns a boolean value indicating whether any packets were
     /// processed or emitted, and thus, whether the readiness of any socket might
     /// have changed.
+    ///
+    /// # Note
+    /// If this function returns true, there may still be packets to be transmitted or received.
+    /// Users should should perform other tasks before calling this function again. If executed in
+    /// a loop, its possible that packets can arrive faster than they can be processed, which could
+    /// cause an infinite loop.
```

Comment:
```suggestion
    /// This function performs a bounded amount of work per call to avoid exhausting
    /// starving other tasks of CPU time. If it returns true, there may still be packets
    /// to be received or transmitted. Depending on system design, calling this function
    /// in a loop may cause a denial of service: it is safe if packets can be processed
    /// faster than they arrive, and unsafe otherwise.
```